File: leetcode/1822 Sign of the Product of an Array/solution.go
func arraySign(nums []int) int { // 仅处理符号变化即可，不需要作乘积！！！
    sign := 1
    for _, num := range nums {
        if num == 0 {
            return 0
        } else if num < 0 {
            sign *= -1
        }
    }
    return sign
}

File: leetcode/1822 Sign of the Product of an Array/solution.java
class Solution {
    public int arraySign(int[] nums) { // 仅处理符号变化即可，不需要作乘积！！！
        int sign = 1;
        for (int num : nums) {
            if (num == 0) return 0;
            if (num < 0) sign *= -1;
        }
        return sign;
    }
}